src/app/home/work-item-widget/work-item-widget.component.html

```diff
@@ -1,44 +1,58 @@
 <div class="create-work-item-widget card-pf card-f8">
   <div class="card-pf-heading-bg">
     <div class="card-pf-heading">
-      <h2 class="card-pf-title">My work items</h2>
+      <h2 class="card-pf-title">My work items
+      <span *ngIf="spaces.length !== 0">
```

Comment:
This isn't what is shown in the mockup I worked from? In fact, the select menu covers the card title.

<img width="446" alt="screen shot 2017-04-26 at 10 26 11 pm" src="https://cloud.githubusercontent.com/assets/17481322/25465328/8ddef02e-2acf-11e7-831d-28a67da9181c.png">